.github/workflows/master-pushed.yml

```diff
@@ -0,0 +1,26 @@
+on:
+  push:
+    branches: [master]
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - uses: actions/setup-node@v4
+        with:
+          node-version-file: package.json
+          cache: npm
+          registry-url: 'https://registry.npmjs.org'
+
+      - run: npm ci
+
+      - run: npm run test
+
+      - run: npx auto-npm-version
```

Comment:
This command will run `npm version` which trigger `npm run-script version` then `npm publish` and finally `git push` the newly created tag.